src/libs/client.js

```diff
@@ -509,3 +509,93 @@ export const connectOidcClient = async (client, oidcCode) => {
     state: STATE_CONNECTED
   }
 }
+
+export const connectMagicLinkClient = async (client, magicCode) => {
+  const stackClient = client.getStackClient()
+
+  let oauthOptions = stackClient.oauthOptions
+  const data = {
+    magic_code: magicCode,
+    client_id: oauthOptions.clientID,
+    client_secret: oauthOptions.clientSecret,
+    scope: '*'
```

Comment:
why do we need to pass the `scope`?

Reply:
Seems that this is not documented on the stack's side https://github.com/cozy/cozy-stack/pull/3872/files#diff-d9cbd97d17d60b9a415f168c988e7af55ab4d6945e24b0fb570a0577eff6b884R294

Reply:
Good catch, I assumed it was the same signature as for the OIDC endpoint, I should have double-checked that part.

As this code has been moved in another file in other PRs, I did the fix in this PR: https://github.com/cozy/cozy-flagship-app/pull/737/commits/3ba803f0e4bd002462957b8a79b68abb391bbdd3